.gitignore

```diff
@@ -4,3 +4,6 @@
 /.idea/libraries
 .DS_Store
 /build
+.idea/
+android-authenticator.iml
```

Comment:
Yes. this would be much better and include all files. I will include this in my next PR.